website/docs/spec/naming.md

```diff
@@ -14,31 +14,32 @@ datasources.
+| Redshift                      | Warehouse                            | `redshift://{cluster_identifier}.{region_name}:{port}`         | `{database}.{schema}.{table}`                                                |
+| Snowflake                     | Warehouse                            | `snowflake://{organization name}-{account name}`               | `{database}.{schema}.{table}`                                                |
+| Trino                         | Warehouse                            | `trino://{host}:{port}`                                        | `{catalog}.{schema}.{table}`                                                 |
+| ABFSS (Azure Data Lake Gen2)  | Data lake                            | `abfss://{container name}@{service name}.dfs.core.windows.net` | `{path}`                                                                     |
+| DBFS (Databricks File System) | Distributed file system              | `dbfs://{workspace name}`                                      | `{path}`                                                                     |
+| GCS                           | Blob storage                         | `gs://{bucket name}`                                           | `{object key}`                                                               |
+| HDFS                          | Distributed file system              | `hdfs://{namenode host}:{namenode port}`                       | `{path}`                                                                     |
+| Kafka                         | distributed event streaming platform | `kafka://{bootstrap server host}:{port}`                       | `{topic}`                                                                    |
+| Local file system             | File system                          | `file`                                                         | `{path}`                                                                     |
+| Remote file system            | File system                          | `file://{host}`                                                | `{path}`                                                                     |
+| S3                            | Blob Storage                         | `s3://{bucket name}`                                           | `{object key}`                                                               |
+| WASBS (Azure Blob Storage)    | Blob Storage                         | `wasbs://{container name}@{service name}.dfs.core.windows.net` | `{object key}`                                                               |
+| PubSub                        | distributed event streaming platform | `pubsub`                                                       | `topic:{projectId}:{topicId}` or `subscription:{projectId}:{subscriptionId}` |
```

Comment:
tiny change but distributed -> Distributed 
😄

Reply:
now i see that its like that for kafka too...

Reply:
Could you make both PubSub and Kafka start with `Distributed`? for consistency sake

Reply:
Correct, I copy/pasted. Fixed in both places